README.md

```diff
@@ -145,6 +145,8 @@ If all else fails please file an issue on the [github project](https://github.co
 
 `void *iso_realloc(void *p, size_t size)` - Equivalent to `realloc`. Reallocates a new chunk, if necessary, to be size bytes big and copies the contents of p to it.
 
+`void *iso_realloc(void *p, size_t size)` - Equivalent to `reallocarray`. In the same principles as `iso_realloc`, reallocates a new chunk but for an array of nmemb elements of size bytes and in addition check for possible size overflow.
```

Comment:
This should be `void *iso_reallocarray`.

Reply:
ah yes true :)